integration/mesos/pom.xml

```diff
@@ -68,16 +68,6 @@
       <artifactId>alluxio-core-server-worker</artifactId>
       <version>${project.version}</version>
     </dependency>
-    <dependency>
```

Comment:
Like our server modules, the Mesos module has no compile-time dependency on UFS, only a runtime dependency. The Mesos module is compiled into the server uber-jar, and this jar is used to run the Mesos integration. The uber-jar contains all UFS dependencies.